doc/source/contributing.rst

```diff
@@ -286,13 +286,21 @@ Some other important things to know about the docs:
   overviews per topic together with some other information (what's new,
   installation, etc).
 
-- The docstrings follow the **Numpy Docstring Standard** which is used widely
+- The docstrings follow the **Numpy Docstring Standard**, which is used widely
   in the Scientific Python community. This standard specifies the format of
   the different sections of the docstring. See `this document
   <https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`_
   for a detailed explanation, or look at some of the existing functions to
   extend it in a similar manner.
 
+- The ``.rst`` files are used to automatically generate Markdown and HTML versions
+  of the docs. For this reason, please do not edit ``CONTRIBUTING.md`` directly, but 
```

Comment:
I would put this in a note directive
and be clear the generation of markdown is only for contributing.md

a